src/Stratis.Bitcoin.Tests/P2P/PeerAddressManagerTests.cs

```diff
@@ -168,7 +168,7 @@ public void PeerAddressManager_AttemptThresholdTimeNotReached_DoNotReset()
             Assert.Null(savedPeer.LastConnectionSuccess);
             Assert.Null(savedPeer.LastConnectionHandshake);
             Assert.Null(savedPeer.LastSeen);
-            Assert.Equal("127.0.0.1", savedPeer.Loopback.ToString());
+            Assert.Equal("::ffff:127.0.0.1", savedPeer.Loopback.ToString());
```

Comment:
How did this ever pass?

Reply:
I dont think it did!

Reply:
@noescape00 made changes here so loop back is now mapped to IPv6